### ISSUE TITLE
grails.web.servlet.mvc.GrailsParameterMap cannot be cast to grails.views.api.http.Parameters

### ISSUE THREAD

**Issue by lifeisfoo** [grails/grails-views#59]
Hi, I'm using views-json 1.1.0RC1 with grails 3.1.9 and I'm receiving an error when trying to render a view with the params helper:
```
json {
    offset params.int('offset', 0) 
}
```
This is the exception:
```
WARN org.eclipse.jetty.servlet.ServletHandler - 
org.springframework.web.util.NestedServletException: Request processing failed; nested exception is grails.views.ViewRenderException: Error rendering view: grails.web.servlet.mvc.GrailsParameterMap cannot be cast to grails.views.api.http.Parameters
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:979)
	//... cut
Caused by: grails.views.ViewRenderException: Error rendering view: grails.web.servlet.mvc.GrailsParameterMap cannot be cast to grails.views.api.http.Parameters
	at grails.views.AbstractWritableScript.writeTo(AbstractWritableScript.groovy:33)
	at grails.views.mvc.GenericGroovyTemplateView.renderMergedOutputModel(GenericGroovyTemplateView.groovy:68)
	at org.springframework.web.servlet.view.AbstractView.render(AbstractView.java:303)
	//... cut
Caused by: java.lang.ClassCastException: grails.web.servlet.mvc.GrailsParameterMap cannot be cast to grails.views.api.http.Parameters
	at ms_clienti_application_list_gson$_run_closure1$_closure2.doCall(ms_clienti_application_list_gson:9)
	at ms_clienti_application_list_gson$_run_closure1$_closure2.call(ms_clienti_application_list_gson)
	at grails.plugin.json.builder.StreamingJsonBuilder$StreamingJsonDelegate.cloneDelegateAndGetContent(StreamingJsonBuilder.java:719)
	at grails.plugin.json.builder.StreamingJsonBuilder$StreamingJsonDelegate.cloneDelegateAndGetContent(StreamingJsonBuilder.java:711)
	at grails.plugin.json.builder.StreamingJsonBuilder$StreamingJsonDelegate.call(StreamingJsonBuilder.java:606)
	at grails.plugin.json.builder.StreamingJsonBuilder$StreamingJsonDelegate.invokeMethod(StreamingJsonBuilder.java:472)
	at ms_clienti_application_list_gson$_run_closure1.doCall(ms_clienti_application_list_gson:7)
	at ms_clienti_application_list_gson$_run_closure1.call(ms_clienti_application_list_gson)
	at grails.plugin.json.builder.StreamingJsonBuilder$StreamingJsonDelegate.cloneDelegateAndGetContent(StreamingJsonBuilder.java:719)
	at grails.plugin.json.builder.StreamingJsonBuilder$StreamingJsonDelegate.cloneDelegateAndGetContent(StreamingJsonBuilder.java:711)
	at grails.plugin.json.builder.StreamingJsonBuilder.call(StreamingJsonBuilder.java:211)
	at grails.plugin.json.view.JsonViewTemplate.json(JsonViewTemplate.groovy:76)
	at ms_clienti_application_list_gson.run(ms_clienti_application_list_gson:5)
	at grails.plugin.json.view.JsonViewTemplate.doWrite(JsonViewTemplate.groovy:35)
	at grails.views.AbstractWritableScript.writeTo(AbstractWritableScript.groovy:30)
```

**Comment by jameskleeh** [comment]
I included a test for this on 3.1.9 and wasn't able to reproduce the issue

https://github.com/grails/grails-views/blob/c6f3b2c7cb1fd118e67e552561a1d8059b23537f/functional-tests/src/integration-test/groovy/functional/tests/BookSpec.groovy#L104

Please try to reproduce on `1.1.0.BUILD-SNAPSHOT` and if you encounter the same problem, upload an example project.

**Comment by jameskleeh** [comment]
Closing this as no feedback provided. Create a new issue with an example project if this is still an issue you are facing
